packages/web-components/fast-components/src/text-field/text-field.styles.ts

```diff
@@ -38,20 +40,21 @@ export const textFieldStyles = (context, definition) =>
         ${display("inline-block")}
         ${inputStyles(context, definition, ".root")}
         ${inputStateStyles(context, definition, ".root")}
+
         .root {
             display: flex;
             flex-direction: row;
+            gap: calc(${densityComponentHorizontalBetweenPadding} * 1px);
```

Comment:
This could bite us in terms of browser versions supported. Many of the things we are leveraging in terms of modern browsers either have fallback support where not implemented yet (adopted stylesheets) or in the case of things like CSS grid were more robustly included in earlier versions than this. This moves the needle further back and could create issues with commitments we've made. Probably something to discuss.

Reply:
Would it be worth it to use gap for browsers that support it and add a matchMedia behavior as a stopgap measure until wider support is available?

This way it would end up being less of a refactor when gap is better supported.

Reply:
It actually looks like gap is just about there on support.

https://developer.mozilla.org/en-US/docs/Web/CSS/gap#browser_compatibility